rpcs3/rpcs3qt/rpcn_settings_dialog.cpp

```diff
@@ -358,6 +390,27 @@ void rpcn_account_dialog::refresh_combobox()
 	cbx_servers->setCurrentIndex(index);
 }
 
+void rpcn_account_dialog::refresh_countrybox()
+{
+	g_cfg_rpcn.load();
+	const auto vec_countries = g_cfg_rpcn.get_countries();
+	const auto cur_country = g_cfg_rpcn.get_country();
+	int i = 0, index = 0;
+
+	cbx_countries->clear();
+
+	for (const auto& [cnty, code] : vec_countries)
+	{
+		cbx_countries->addItem(QString::fromStdString(cnty), QString::fromStdString(code));
+		if (cur_country == code)
+			index = i;
+
+		i++;
+	}
+	cbx_countries->setCurrentIndex(index);
```

Comment:
You can remove all this index bloat by just writing:
```suggestion
	cbx_countries->setCurrentIndex(cbx_countries->findData(QString::fromStdString(cur_country)));
```